cernopendata_client/cli.py

```diff
@@ -362,3 +364,27 @@ def verify_files(server, recid):
         msg_type="success",
         msg="Success!",
     )
+
+
+@cernopendata_client.command()
+@click.option("--path", type=click.STRING, help="EOS PUBLIC Path")
+def list_directory(path):
+    """List file in a directory.
+
+    Select a EOSPUBLIC path and return the list of files
+    belonging to this directory and its subdirectories.
+
+    \b
+    Examples:
+      $ cernopendata-client list-directory /eos/opendata/cms/validated-runs/Commissioning10
```

Comment:
The example does not correspond to the current behaviour.

Example says:

```console
$ cernopendata-client list-directory /eos/opendata/cms/validated-runs/Commissioning10
```

but one is expected to use:

```console
$ cernopendata-client list-directory --path /eos/opendata/cms/validated-runs/Commissioning10
```

The example is more intuitive. Please amend to rather use argument than option.

```